CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ add_library(framework SHARED
   src/fire/Conditions.cxx
   src/fire/RandomNumberSeedService.cxx
   )
-target_link_libraries(framework PUBLIC logging version exception config factory io)
+target_link_libraries(framework PUBLIC logging version exception config factory io Boost::boost)
```

Comment:
This additional link _shouldn't_ be needed since `Boost::boost` is already included in `config` and `factory` as a `PUBLIC` dependency. Did you add it after seeing that it didn't work without it? Or did you add it because `framework` uses some boost headers occasionally?